spec.html

```diff
@@ -6235,6 +6236,63 @@
         1. If _result_ is an abrupt completion, perform HostReportErrors(&laquo; _result_.[[Value]] &raquo;).
     </emu-alg>
   </emu-clause>
+
+  <emu-clause id="sec-agents">
+    <h1>Agents</h1>
+
+    <p> An <dfn id="agent">agent</dfn> comprises a set of ECMAScript execution contexts, an execution context stack, a running execution context, a set of named job queues, an <dfn id="agent-record">Agent Record</dfn>, and an <dfn id="executing-thread">executing thread</dfn>.  Except for the executing thread, the constituents of an agent belong exclusively to that agent. </p>
+
+    <p> An agent's executing thread executes the jobs in the agent's job queues on the agent's execution contexts independently of other agents, except that an executing thread may be used as the executing thread by multiple agents if none of the agents sharing the thread have an Agent Record whose [[CanBlock]] property that is ~true~. </p>
+
+    <emu-note>
+      <p> Some web browsers share an executing thread across the documents of a browser window, for example. </p>
+    </emu-note>
+
+    <p> While an agent's executing thread executes the jobs in the agent's job queues, the agent is the <dfn id="surrounding-agent">surrounding agent</dfn> for the code in those jobs.  The code uses the surrounding agent to access the specification level execution objects held within the agent: the running execution context, the execution context stack, the named job queues, and the Agent Record's fields. </p>
+
+    <emu-table id="table-agent-record" caption="Agent Record Fields">
+      <table>
+        <tbody>
+          <tr>
+            <th> Field name </th>
+            <th> Value </th>
+            <th> Meaning </th>
+          </tr>
+          <tr>
+            <td> [[LittleEndian]] </td>
+            <td> Boolean </td>
+            <td> The default value computed for the <em>isLittleEndian</em> parameter when it is needed by the algorithms GetValueFromBuffer and SetValueInBuffer.  The choice is implementation dependent and should be the alternative that is most efficient for the implementation. Constant. </td>
```

Comment:
Or maybe I'm being overly cautious.  It's necessary for integrity that the field not take on more than one value at run-time (for the given agent).  It's also clear from the spec text that nothing sets this field, but that doesn't really prevent the implementation from doing so, hence the "constant" annotation.  But I will attempt to rephrase.